api/jobs/dto/full-source-file-dto.ts

```diff
@@ -2,6 +2,7 @@ import { SourceFileDto } from "./source-file-dto";
 
 interface FullSourceFileDto extends SourceFileDto {
```

Comment:
@PavelLoparev  I would recommend checking where we use it. Maybe we can eliminate it. I hope jobs API is consistent and we have 3 fields everywhere (uri, name, fileUid).

Reply:
Thx for your contribution. Let's wait for another review from Smartling dev.

Reply:
@dimitrystd unfortunately jobs api returns different file dtos from different endpoints. that's why we have `SourceFileDto` and `FullSourceFileDto` (at least it's true for the time these dtos were introduced in this sdk, I didn't check jobs api recently).

Reply:
Ok, https://api-reference.smartling.com/#tag/Jobs is a bit outdated on responses dto. I'll notify jobs api team about that.

Reply:
@PavelLoparev I contacted them already